kart/output_util.py

```diff
@@ -190,7 +190,7 @@ def dump_json_output(
 
     fp = resolve_output_path(output_path)
 
-    highlit = json_style == "pretty" and can_output_colour(fp)
+    highlit = can_output_colour(fp)
```

Comment:
What's the reasoning for this change? Prettier json-lines output?

Reply:
yep. optional I suppose. It still doesn't colour the output if it's not going to a TTY though